packages/cli/src/License.ts

```diff
@@ -51,18 +51,19 @@ export class License {
 			return;
 		}
 
-		const isMainInstance = instanceType === 'main';
+		const isLeaderMain = instanceType === 'main' && this.orchestrationService.isLeader;
+
 		const server = config.getEnv('license.serverUrl');
-		const autoRenewEnabled = isMainInstance && config.getEnv('license.autoRenewEnabled');
-		const offlineMode = !isMainInstance;
+		const autoRenewEnabled = isLeaderMain && config.getEnv('license.autoRenewEnabled');
```

Comment:
I think only `autoRenewEnabled` needs to change, the others can stay as they are probably. 

- saving a certificate only happens on renew, so if not renewing, it should be fine
- feature changes will happen following a change to license, which afaik only happens when license changes or leader asks followers for updates
- collectUsageMetrics is probably also something that should not pose any issues

The problem is then that we need to change the `autoRenewEnabled` setting whenever a follower becomes a leader, right?